lib/compat/wordpress-6.2/site-editor.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Compatibility fixes for WP 6.1 and Site Editor.
+ *
+ * It can be removed after the minimum required WP version is 6.2
+ *
+ * @package WordPress
+ */
+
+/**
+ * Preload theme variations in site editor.
+ *
+ * @param string[]                $preload_paths  Array of paths to preload.
+ * @param WP_Block_Editor_Context $block_editor_context The current block editor context.
+ *
+ * @return string[] Preload paths.
+ */
+function filter_block_editor_rest_api_preload_paths( $preload_paths, $block_editor_context ) {
+	if ( 'core/edit-site' === $block_editor_context->name ) {
```

Comment:
```suggestion
	if ( isset( $block_editor_context->name ) && 'core/edit-site' === $block_editor_context->name ) {
```

The context name property was introduced in WP 6.0. Let's use an extra check for the minimum required 5.9.